ted/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -171,14 +177,22 @@ private void bindBrownoutActions(){
     }
   }
 
+   /**
+   * A method to init all the data logging
+   */
+  private void initializeDataLogging() {
```

Comment:
This method should come after the next one in alphabetical order.

Reply:
swapped